snippets/legv8.json

```diff
@@ -0,0 +1,185 @@
+{
+  "set register to": {
+    "prefix": "sreg",
+    "body": ["ADD ${1:dest reg}, XZR, $2", "$3"]
+  },
+  "increment register": {
+    "prefix": "increg",
+    "body": ["ADDI ${1:dest reg}, ${1}, #1", "$2"]
+  },
+  "decrement register": {
+    "prefix": "decreg",
+    "body": ["SUBI ${1:dest reg}, ${1}, #1", "$2"]
+  },
+  "copy register content to another register": {
+    "prefix": "copy",
+    "body": ["ORR ${1:dest reg}, XZR, ${2:reg to copy from}", "$3"],
+    "description": "equivalent to 'MOV Xd, Xm'"
+  },
+  "clear register content": {
+    "prefix": "clearReg",
+    "body": ["EOR ${1:target reg}, $1, $1", "$2"],
+    "description": "equivalent of CLEAR. It clears the content of a target register"
+  },
+  "increment offset by 8 bits": {
+    "prefix": "inc8",
+    "body": ["ADDI ${1:offset to increment}, $1, #8"],
+    "description": "incrementing according to offset allignment standard, in order to properly traverse through an array"
+  },
+
+  "store 2 registers to save on stack": {
+    "prefix": "st2",
+    "body": [
+      "//storing registers in stack before changing them",
```

Comment:
all comments should have a space after them.
`//text` should change to `// text`

Reply:
Good call